### DISCUSSION TITLE
Python error

### DISCUSSION THREAD

**Discussion by ArcadiaBaes** [Mustard2/MustardUI#196] [Help]
Im trying to add a color ramp node to the ui, but everytime i get this error.
![image](https://github.com/user-attachments/assets/bc322f24-48ef-4a34-a741-56c600d662f7)
As i dont know phyton, what am i supposed to edit to make it work?
![image](https://github.com/user-attachments/assets/0c03f639-f9bb-4437-81e5-50da7f4466be)

**Comment by Mustard2** [comment]
Hi! Where is the error thrown? Do you have the whole modified code I can check?

**Comment by ArcadiaBaes** [comment]
its seems the error is in line 97?

**Reply by Mustard2** [reply]
Ah sorry for some reasons the first image didn't load :D
I have to check it

**Comment by ArcadiaBaes** [comment]
thank you!

**Comment by Mustard2** [comment]
Openened an issue to keep track of the problem: https://github.com/Mustard2/MustardUI/issues/199

Closing this, please open the linked Issue for further discussion
